src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
```

Comment:
At this point, we can some construction improvements:
1. Use Util class because we don't have state there https://www.baeldung.com/java-helper-vs-utility-classes
2. Bring this logic into the Color enum like
 `public static Color random() {
        int index = new Random().nextInt(Colors.values().length);
        return Colors.values()[index];
    }` I would prefer the 2nd option because it is more natural